compiler/tests-common/tests/org/jetbrains/kotlin/codegen/AbstractCheckLocalVariablesTableTest.java

```diff
@@ -82,17 +85,31 @@ protected void doTest(@NotNull String ktFileName) throws Exception {
         ClassReader cr = new ClassReader(outputFile.asByteArray());
         List<LocalVariable> actualLocalVariables = readLocalVariable(cr, methodName);
 
-        KotlinTestUtils.assertEqualsToFile(ktFile, text.substring(0, text.indexOf("// VARIABLE : ")) + getActualVariablesAsString(actualLocalVariables));
+        List<String> actual = getActualVariablesAsString(actualLocalVariables);
```

Comment:
It's best to keep old compare logic cause it's allow FileComparisonFailure allows easily update failed testdata

Reply:
Old compare logic relies on the order of the variables which is not guaranteed in the two backends. I rolled back changes in non-ir tests and moved them to ir part as suggested above.